.vscode/extensions.json

```diff
@@ -22,6 +22,7 @@
     "vmware.vscode-boot-dev-pack",
     "vscjava.vscode-gradle",
     "vscjava.vscode-java-pack",
-    "webhint.vscode-webhint"
+    "webhint.vscode-webhint",
+    "ms-playwright.playwright"
```

Comment:
The file `.vscode/exntesions.json` will be removed in the near future because the dev container already defined the extensions (duplicated content). I have added the extension `ms-playwright.playwright` to the definition of the dev container in #2025